modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -25,10 +25,12 @@
     <template #main>
       <AposModalBody>
         <template #bodyHeader>
-          <AposMediaManagerToolbar
-            v-if="!!media.length"
-            :checked="checked" :media="media"
+          <AposDocsManagerToolbar
+            v-if="!!items.length"
+            :checked="checked" :media="items"
+            :selected-state="selectAllState"
```

Comment:
This uses selectAllState now instead of passing in media and checked arrays to be processed.

Reply:
selectAllState is in the mixin